gufe/settings/models.py

```diff
@@ -109,23 +103,30 @@ class ThermoSettings(SettingsBaseModel):
        possible.
     """
 
-    temperature: FloatQuantity["kelvin"] = Field(None, description="Simulation temperature, default units kelvin")
-    pressure: FloatQuantity["standard_atmosphere"] = Field(
-        None, description="Simulation pressure, default units standard atmosphere (atm)"
-    )
+    temperature: KelvinQuantity | None = Field(None, description="Simulation temperature in kelvin)")
+    pressure: AtmQuantity | None = Field(None, description="Simulation pressure in standard atmosphere (atm)")
     ph: PositiveFloat | None = Field(None, description="Simulation pH")
     redox_potential: float | None = Field(None, description="Simulation redox potential")
```

Comment:
I've never actually performed simulations where `redox_potential` is set. Does it not typically have a unit attached?

Reply:
@IAlibay do you have any insight here?

Reply:
Redox potential is measured in volts, usually folks will pass through a value in mV.

Reply:
thanks @IAlibay! Does this mean you'd prefer the values get converted from `mV` to `volt`, or is it preferable to keep the inputs as `mV`?

Reply:
Keep the input as mV, that's the human readable version.

Reply:
I updated it yesterday to be mV (see here https://github.com/OpenFreeEnergy/gufe/blob/dd3bc616803c133351f91a9838d1f88f1f1b3346/gufe/settings/models.py#L110-L111)